ionic2/app/components/map/map.component.ts

```diff
@@ -1,13 +1,16 @@
 import { Component, ViewChild } from '@angular/core';
+import { PokeSighting } from '../../models/poke-sighting';
+import { PokePrediction } from '../../models/poke-prediction';
+import { PokeMob } from '../../models/poke-mob';
+import { PokePOI } from '../../models/poke-poi';
 
-let PokeMap = require('pokemap-2');
```

Comment:
With pokemap-1 the feature does not work as they haven't implemented the click event yet. So I need to change the map in order to make it work.